system/lib/wasmfs/file.h

```diff
@@ -65,7 +65,9 @@ class File : public std::enable_shared_from_this<File> {
     return (ino_t)this;
   }
 
-  backend_t getBackend() { return backend; }
+  backend_t getBackend() const { return backend; }
+
+  bool getSeekable() const { return seekable; }
```

Comment:
How come we didn't already have this for stdin/stdout which are not seekable either?

Reply:
Good point! I marked those non-seekable now. I think we just didn't notice this before as we didn't reach a test that checked that yet.

Reply:
Can we name this `isSeekable` since it's used as a predicate?

Reply:
Sure, fixing.